extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```
@@ -231,5 +202,91 @@ describes.realWin(
         ).to.have.been.calledOnce;
       });
     });
+
+    it('test validate required fields for config', async () => {
```

Nit: 'should fail config validation because a required config value is missing'